pkg/piecestore/rpc/client/pieceranger_test.go

```diff
@@ -147,7 +153,14 @@ func TestPieceRangerSize(t *testing.T) {
 
 		ctx := context.Background()
 
-		c, err := NewCustomRoute(route, node.IDFromString("test-node-id-1234567"), 32*1024, priv)
+		_node := &pb.Node{
```

Comment:
Use `node` or `nodeInfo` or `target` or `destination` rather than the underscore prefix.